addons/website/views/snippets/s_masonry_block.xml

```diff
@@ -222,19 +204,19 @@
             style="grid-area: 1 / 1 / 11 / 4; z-index: 1;">
             <img src="/web/image/website.s_masonry_block_default_image_1" class="img img-fluid mx-auto" alt=""/>
         </div>
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-10 text-center o_cc o_cc4" data-name="Block"
-            style="grid-area: 1 / 4 / 11 / 7; z-index: 2;">
-            <h3>A great title</h3>
-            <p>And a great subtitle</p>
+        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-10 text-center o_draggable o_colored_level o_cc o_cc2" data-name="Block" style="grid-area: 1 / 4 / 11 / 7; z-index: 2; position: relative;" data-oe-shape-data="{&quot;shape&quot;:&quot;web_editor/Floats/05&quot;,&quot;flip&quot;:[],&quot;showOnMobile&quot;:false,&quot;shapeAnimationSpeed&quot;:&quot;0&quot;,&quot;animated&quot;:&quot;true&quot;}">
+            <div class="o_we_shape o_web_editor_Floats_05 o_we_animated"/>
```

Comment:
I'm not convinced about this shape 🤔 Here's a test using `Rainy 03`, what do you think ?
![Capture d’écran 2024-07-19 à 17 33 38](https://github.com/user-attachments/assets/e3e32646-236c-4b1e-ae76-90743a50da05)